### ISSUE TITLE
Missing elements in style.css

### ISSUE THREAD

**Issue by RonWillemse** [copyblogger/genesis-sample#14]
why are ::-moz-selection and ::selection removed from the genesis sample develop theme?

**Comment by RRWD** [comment]
Because they gave a validation parse error on the W3C CSS validator.
And if your theme doesn't parse, it's not WCAG 2 valid.

**Comment by GaryJones** [comment]
It's looking like it is going to be in CSS4: http://www.w3.org/TR/css-pseudo-4/#selectordef-selection

Have you got a reference for the part of WCAG that says the CSS should be valid please?

**Comment by RRWD** [comment]
Looked it up and checked with Iacobien Riezebosch, who is the leading Dutch web accessibility expert. Parsing is only required for HTML in WCAG 2 AA. It is an item of dispute with the Dutch accessibility inspection institutes. 
But I think the CSS should always pars without errors for good practice. And as I said with the PR:
It's not really required but will save you support questions if people run their theme through validations software and get errors.

**Comment by GaryJones** [comment]
Anyone taking it upon themselves should also be aware that the validator is not perfect, and that as in this case, the document specifications are paving the cow paths since all of the major players support `::selection` already.

You've only got to look at the use of vendor prefixed properties to know that real world and validators don't always mix nicely, and yet accessibility is not generally hindered by the use of those vendor prefixed properties.

**Comment by RRWD** [comment]
In the support mail I get for Leiden I noticed that people are worried about not validating CSS and think the a11y is wrong. And since it's not a primary feature I thought better leave it out.
But by all means, if you need it put it back, as you said it doesn't hinder a11y, but a green validate sign is always nice to show to your customer.

**Comment by carlomanf** [comment]
I lean towards agreeing with Rian, because it's a low-impact change. From an ugliness perspective, I think this change is trivial compared to the underlining of headings.

**Comment by RonWillemse** [comment]
also the element blockquote::before is missing: font-size: 3.0rem;
